Client-Setup-Windows.md

```diff
@@ -96,10 +96,12 @@ After confirming that your client can connect to a server, you can use one of th
 
 3. Alter the `[ashita.boot]` section in `localhost.ini` to look like this:
 
+   **Note: Ashita comes bundled with it's own bootloader (pol.exe). To connect to the latest version of the server you will need to use the most current version of xiloader. You will need to adjust your boot ini file to point to xiloader.**
+
    ```ini
    [ashita.boot]
-   ; Private Server Usage
-   file        = .\\bootloader\\pol.exe
+   ; Private Server Usage, using xiloader.exe instead of pol.exe
+   file        = .\\bootloader\\xiloader.exe
```

Comment:
with the shear number of people I've had to deal with that drop their ashita (and thus is bootloader folder) in poorly chosen locations, renamed their exe's to be compatible with things that expect "pol.exe" in the process list and misc. other issues that prove the world continues to churn out more powerful idiots the more you idiot proof, I think I'd rather the example be `C:\some folder\xiloader.exe` rather than directing them to use the same location as an existing executable that came with ashita.

But that's just my 2 cents. carry on as you wish. As it is a **majority of them do not read this page** and when I tell them their loader is outdated they download it, double click, and insist they "installed the update".